Domains/Core/BisCore.ecschema.xml

```diff
@@ -566,14 +565,14 @@
 
     <ECEntityClass typeName="GeometryDetailRecord" modifier="Abstract" displayLabel="Geometry Detail Record" description="An abstract InformationRecordElement that holds geometry for use by the 'feature tree' stored in the `GeometricElement3d.GeometryOperations` property of its parent Element.">
         <BaseClass>InformationRecordElement</BaseClass>
-        <ECProperty propertyName="GeometryStream" typeName="binary" extendedTypeName="GeometryStream" displayLabel="Geometry Stream" description="Binary stream used to persist the geometry of this bis:Element.">
+        <ECProperty propertyName="GeometryMap" typeName="binary" extendedTypeName="GeometryMap" displayLabel="Geometry Map" description="Binary stream used to persist the geometries of this bis:Element.">

```

Comment:
Need a .remarks.md file documenting the format or linking to its documentation. Should also link to docs for the API for working with this stuff.

Reply:
Added the remarks, the API is not published to public registry yet.